bullet_train-themes-light/app/views/showcase/previews/partials/_team.html.erb

```diff
@@ -0,0 +1,22 @@
+<% showcase.sample "Basic" do %>
+  <ul>
+    <%= render "shared/team" do |cell| %>
+      <% cell.name "Name" %>
+    <% end %>
+  </ul>
+<% end %>
+
+<% showcase.sample "With Memberships" do %>
+  <ol>
+    <%= render "shared/team", memberships: Membership.all do |cell| %>
+      <% cell.name "Title" %>
+    <% end %>
+  </ol>
+<% end %>
```

Comment:
I'm a little unsure of if we can rely on Membership always being defined, but lets see how this goes. We can always wrap it later ala

```erb
<% if defined?(Membership.all) %>
  <% showcase.sample "With Memberships" do %>
    <ol>
      <%= render "shared/team", memberships: Membership.all do |cell| %>
        <% cell.name "Title" %>
      <% end %>
    </ol>
  <% end %>
<% end %>
```